Samples/ObservableScreen/Sources/MultiCounterView.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+import Perception
+import SwiftUI
+import ViewEnvironment
+import Workflow
+import WorkflowSwiftUI
+
+struct MultiCounterView: View {
+    @Perception.Bindable var store: Store<MultiCounterModel>
+
+    var body: some View {
+        WithPerceptionTracking {
+            let _ = print("Evaluated MultiCounterView body")
+            VStack {
+                Text("Multi Counter Demo")
+                    .font(.title)
+
+                controls
+
+                if let maxCounter = store.maxCounter {
+                    CounterView(store: maxCounter, key: "max")
+                }
+
+                ForEach(
+                    Array(store.counters.enumerated()),
+                    id: \.element.id
+                ) { index, counter in
```

Comment:
I'm curious when you do the `ForEach` on `counters` which is `CounterWorkflow.Model` what translates that to a `Store<CounterWorkflow.Model>` when you get inside the loop?

Reply:
I do really like how you don't need to call to scope the store to a subset of the store when creating a child view.